third_party/stdlib/re.py

```diff
@@ -0,0 +1,30 @@
+#
+# Secret Labs' Regular Expression Engine
```

Comment:
Stuff from pypy needs to be in a different subdir of third_party (maybe third_party/pypy).